### ISSUE TITLE
get.rvm.io Failing Consistently

### ISSUE THREAD

**Issue by gnossen** [grpc/grpc#27585]
```
++ rvm get stable
Downloading https://get.rvm.io
Could not download rvm-installer, please report to https://github.com/rvm/rvm/issues
```

This is currently affecting the [Artifact Build MacOS](https://source.cloud.google.com/results/invocations/127383bd-a21b-4803-91d5-b112e65422fa/targets) and [Basic Test Ruby MacOS](https://source.cloud.google.com/results/invocations/19a925da-1fb3-499d-9a62-0a44c2209a1b/targets) jobs.

@stanley-cheung Please route if you are no longer the appropriate contact for this.

**Comment by drfloob** [comment]
This is being fixed in https://github.com/grpc/grpc/pull/27538

**Comment by gnossen** [comment]
@drfloob Shouldn't the issue be left open until the fix is merged?

**Comment by gnossen** [comment]
Including the string "Fixes #27585" in either the PR description or in a commit message will automatically close this issue when the fix PR is merged.

**Comment by drfloob** [comment]
Sure, that works too.
